lib/ama_layout/notification_set.rb

```diff
@@ -20,7 +21,7 @@ class NotificationSet
     include Enumerable
     attr_accessor :base, :data_store, :key
 
-    delegate :each, :first, :last, :size, :[], :empty?, :any?,  to: :active
+    delegate :each, :first, :last, :size, :[], :empty?, :any?,  to: :all
```

Comment:
Is this delegation still needed if they're acting on `all`?  I might be missing something but it seems like that should be the default.